conda_forge_tick/migrators.xsh

```diff
@@ -166,12 +175,16 @@ class Version(Migrator):
 
     def filter(self, attrs):
         conditional = super().filter(attrs)
-        return bool(conditional  # if archived
-                or not attrs.get('new_version')  # if no new version
-                # if new version is less than current version
-                or VersionOrder(str(attrs['new_version'])) <= VersionOrder(str(attrs['version']))
-                # if PRed version is greater than newest version
-                or VersionOrder(attrs.get('PRed', '0.0.0')) >= VersionOrder(attrs['new_version']))
+        return bool(
+            conditional  # if archived/finished
+            or not attrs.get('new_version')  # if no new version
+            # if new version is less than current version
+            or (VersionOrder(str(attrs['new_version'])) <=
+                VersionOrder(str(attrs['version'])))
+            # if PRed version is greater than newest version
+            or any(VersionOrder(self._extract_version_from_hash(h)) >=
```

Comment:
I think sometimes `h` will be `False`. Then `_extract_version_from_hash` will error.

Reply:
right, so now we default to '0.0.0'